packages/store/src/index.ts

```diff
@@ -5,13 +5,17 @@ import { initI18n, text } from './i18n';
 import type { Service } from './services';
 import { getInstance as getSystemService } from './services/backend-system';
 import { getInstance as getTelemetrySettingService } from './services/telemetry-setting';
+import { getInstance as getApiHubSettingsService } from './services/api-hub';
 import { getDefaultLogger } from './defaults';
 
+export type EnityName = 'system' | 'telemetrySetting' | 'api-hub';
```

Comment:
There's an enum `Entity` that could probably used, but this is also okay as the store module will change soon and become leaner and only contain the data access layer

Reply:
Ok, I leave it like that then and you can change it when you make the store leaner.